From 5c933890126f3597335930681260acab604c0b7e Mon Sep 17 00:00:00 2001
From: Dominik Hassler <hadfl@omniosce.org>
Date: Thu, 8 Feb 2018 17:23:00 +0100
Subject: [PATCH] 9027 Makefiles need to specify C99 mode consistently (missed
 lx_lockd) Reviewed by: Dan McDonald <danmcd@joyent.com> Reviewed by: Jason
 King <jbk@joyent.com> Approved by: Dan McDonald <danmcd@joyent.com>

---
 usr/src/lib/brand/lx/lx_lockd/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/usr/src/lib/brand/lx/lx_lockd/Makefile b/usr/src/lib/brand/lx/lx_lockd/Makefile
index e535a30877..debefdf815 100644
--- a/usr/src/lib/brand/lx/lx_lockd/Makefile
+++ b/usr/src/lib/brand/lx/lx_lockd/Makefile
@@ -30,7 +30,7 @@ ROOTBIN =	$(ROOTBRANDDIR)
 CLOBBERFILES =	$(OBJS) $(ROOTPROG)
 
 CPPFLAGS +=	-I$(SRC)/cmd/fs.d/nfs/lib
-C99MODE =	$(C99_ENABLE)
+CSTD =		$(CSTD_GNU99)
 LDLIBS +=	-lnsl -lsocket
 
 CERRWARN += -_gcc=-Wno-parentheses
-- 
2.21.0

